adb/fastboot.py

```diff
@@ -70,13 +64,15 @@ class FastbootProtocol(object):
   """Encapsulates the fastboot protocol."""
   FINAL_HEADERS = {'OKAY', 'DATA'}
 
-  def __init__(self, usb):
+  def __init__(self, usb, chunk_kb=1024):
     """Constructs a FastbootProtocol instance.
 
-    Arguments:
+    Args:
       usb: UsbHandle instance.
+      chunk_kb: Packet size. For older devices, 4 may be required.
     """
     self.usb = usb
+    self.chunk_kb = chunk_kb
```

Comment:
`_chunk_kb` (`usb` should be `_usb` as well, but feel free to ignore that change)